modules/admin_manual/pages/installation/manual_installation/server_prep_ubuntu_22.04.adoc

```diff
@@ -0,0 +1,378 @@
+
+IMPORTANT: PHP 8.x is currently **not** supported by ownCloud server.
+
+NOTE: The commands and links provided in this description should give you proper hints but we cannot take any responsibility.
+
+== Upgrading from Ubuntu 20.04 to 22.04
+
+Without going into the upgrade details, you can now upgrade directly from Ubuntu 20.04 LTS to 22.04 LTS when using the do-release-upgrade command. For more details see an example description at {ubuntu_upgrade_url}[Upgrade Ubuntu 20.04 to 22.04 LTS using command line]
+
+[source,bash]
+----
+sudo apt update
+----
+
+[source,bash]
+----
+sudo apt upgrade
+----
+
+[source,bash]
+----
+sudo apt dist-upgrade
+----
+
+[source,bash]
+----
+sudo apt autoremove
+
+----
+[source,bash]
+----
+sudo apt install update-manager-core
+----
+
+[source,bash]
+----
+sudo do-release-upgrade
+----
+
+If you have upgraded successfully and had previously installed any versions of PHP, better uninstall all PHP versions and ALL extensions and perform a fresh PHP 7.4 native installation. To do so, stop any services which depend on or use PHP like your Web Server. Then get a list of all PHP related programs and extensions by typing:
+
+[source,bash]
+----
+dpkg -l | grep php | awk '{print $2}' | tr "\n" " "
+----
+
+Disable all extensions listed (see the section
+xref:useful-commands-for-managing-php-extensions[Useful Commands For Managing PHP Extensions])
+
+Then purge them step by step
+
+[source,bash]
+----
+sudo apt purge <package name from the list above>
+----
+
+Alternatively, if you are fine removing all packages listed, you can run:
+
+[source,bash]
+----
+sudo apt purge `dpkg -l | grep php | awk '{print $2}' | tr "\n" " "`
+----
+
+NOTE: Keep a copy of your relevant `php.ini` and other configuration files as reference _before_ you start purging, to quickly reconfigure your PHP 7.4 installation afterwards.
+
+After deinstallation of all versions and extensions of PHP and cleaning up all remnants, you should have an empty `/etc/php` directory. You can proceed like on a fresh installation of Ubuntu 22.04 LTS.
+
+== Clean Ubuntu 22.04 Installation
+
+After you have installed your Ubuntu 22.04 LTS server from scratch, use the following commands to first add the {ondrej-php-url}[ondrej/php] PPA and then install PHP 7.4 and all necessary extensions:
+
+[source,bash]
+----
+sudo add-apt-repository ppa:ondrej/php -y
+----
+
+[source,bash]
+----
+sudo apt update && sudo apt upgrade
+----
+
+[source,bash]
+----
+sudo apt install php7.4 php7.4-intl php7.4-mysql php7.4-mbstring \
+       php7.4-redis php7.4-imagick php7.4-igbinary php7.4-gmp \
+       php7.4-bcmath php7.4-curl php7.4-gd php7.4-zip php7.4-imap \
+       php7.4-ldap php7.4-bz2 php7.4-ssh2 php-phpseclib \
+       php7.4-common php7.4-json php7.4-xml php7.4-dev php7.4-apcu \
+       php7.4-redis libsmbclient-dev php-pear
```

Comment:
To add php7.4-opcache to align with the quick install guide

Reply:
fixed